src/Service.Host/client/src/components/importBooks/ImpBookPrintOut.js

```diff
@@ -0,0 +1,386 @@
+                </Grid>
+                <Grid item xs={4}>
+                    Country: <b>{supplierCountry}</b>
+                </Grid>
+                <Grid item xs={4}>
+                    EEC Member: <b>{eecMember}</b>
+                </Grid>
+
+                <Grid item xs={12}>
+                    Currency: <b>{currency}</b>
+                </Grid>
+                <Grid item xs={12}>
+                    Parcel Number: <b>{parcelNumber}</b>
+                </Grid>
+
+                <Grid item xs={12} className={classes.gapAbove}>
+                    <Grid item xs={12}>
+                        <Typography variant="h6">Invoice Details</Typography>
+                    </Grid>
+                    <TableWithInlineEditing
+                        columnsInfo={[
+                            {
+                                title: 'Invoice Number',
+                                key: 'invoiceNumber',
+                                type: 'text'
+                            },
+                            {
+                                title: 'Invoice Value (Currency)',
+                                key: 'invoiceValue',
+                                type: 'number',
+                                decimalPlaces: 2
+                            }
+                        ]}
+                        content={invoiceDetails ?? [{}]}
+                        updateContent={() => {}}
+                        allowedToEdit={false}
+                        allowedToCreate={false}
+                        allowedToDelete={false}
+                    />
+                </Grid>
+
+                <Grid item xs={12} className={classes.gapAbove} />
+
+                <Grid item xs={4} />
+                <Grid item xs={4}>
+                    Total Import Value (GBP): <b>{totalImportValue}</b>
+                </Grid>
+                <Grid item xs={4} />
+
+                <Grid item xs={12} className={classes.gapAbove} />
+
+                <Grid item xs={9}>
+                    Carrier:
+                    <b>
+                        {carrierId} - {carrierName}
+                    </b>
+                </Grid>
+                <Grid item xs={3}>
+                    Num Cartons: <b>{numCartons}</b>
+                </Grid>
+
+                <Grid item xs={9}>
+                    Transport Code: <b>{transportCode}</b>
+                </Grid>
+                <Grid item xs={3}>
+                    Num Pallets: <b>{numPallets}</b>
+                </Grid>
+
+                <Grid item xs={9}>
+                    Transport Bill Number: <b>{transportBillNumber}</b>
+                </Grid>
+                <Grid item xs={3}>
+                    Weight: <b>{weight}</b>
+                </Grid>
+
+                <Grid item xs={9}>
+                    Transaction Code: <b>{transactionCode}</b>
+                </Grid>
+                <Grid item xs={3}>
+                    Customs Entry Code:{' '}
```

Comment:
linter might have introduced {' '}

Reply:
It did, but I think for once I actually want it in this file so that there are spaces in the print out